lib/zcaps.js

```diff
@@ -27,8 +30,14 @@ exports.delegateCapability = async (
       '"invocationTarget" must be an object that includes a "type".');
   }
   // TODO: Look into requiring an invoker or controller on a zcap
```

Comment:
Has this been "looked into"? Can we remove this TODO yet or convert it to an issue?

Reply:
There's already an open issue on this https://github.com/digitalbazaar/bedrock-profile/issues/57